styles/blocks/button.scss

```diff
@@ -0,0 +1,43 @@
+@use '../utils/mixins.scss' as *;
+
+.button {
+  border-radius: 40px;
+  display: block;
```

Comment:
If the button is an inline element like 'a' or 'button', using display: block will make the button take the whole line, which is not always desired. Consider using 'display: inline-block' if you want the button to keep its block-like properties while staying inline.